static/package.json

```diff
@@ -9,76 +9,76 @@
   "license": "Apache-2.0",
   "main": "index.js",
   "scripts": {
-    "build": "webpack -p --progress --config webpack.config.js",
-    "dev-build": "webpack --progress -d --config webpack.config.js",
+    "build": "webpack --mode=production --progress --config webpack.config.js",
+    "dev-build": "webpack --progress --mode=development --config webpack.config.js",
     "lint": "eslint --fix .",
     "test": "jest --unhandled-rejections=strict",
     "test-lint": "eslint .",
-    "watch": "webpack --progress -d --config webpack.config.js --watch"
+    "watch": "webpack --progress --mode=development --config webpack.config.js --watch"
   },
   "dependencies": {
-    "@babel/core": "^7.11.6",
-    "@babel/preset-env": "^7.11.5",
-    "@babel/preset-react": "^7.10.4",
-    "@types/d3": "^5.7.2",
-    "@types/googlemaps": "^3.39.13",
+    "@babel/core": "^7.12.10",
+    "@babel/preset-env": "^7.12.11",
+    "@babel/preset-react": "^7.12.10",
+    "@types/d3": "^5.16.4",
+    "@types/googlemaps": "^3.40.5",
     "@types/js-cookie": "^2.2.6",
-    "@types/lodash": "^4.14.161",
-    "@types/reactstrap": "^8.5.1",
-    "axios": "^0.19.2",
-    "babel-loader": "^8.1.0",
+    "@types/lodash": "^4.14.165",
+    "@types/reactstrap": "^8.7.2",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.3.2",
+    "axios": "^0.21.0",
+    "babel-loader": "^8.2.2",
     "babel-polyfill": "^6.26.0",
-    "bootstrap": "^4.5.2",
-    "copy-webpack-plugin": "^5.1.2",
-    "css-loader": "^3.6.0",
+    "bootstrap": "^4.5.3",
+    "copy-webpack-plugin": "^7.0.0",
+    "css-loader": "^5.0.1",
     "d3": "^5.16.0",
     "d3-geo": "^1.12.1",
-    "file-loader": "^6.1.0",
+    "file-loader": "^6.2.0",
     "geo-albers-usa-territories": "0.1.0",
-    "html-webpack-plugin": "^4.4.1",
+    "html-webpack-plugin": "^4.5.0",
     "js-cookie": "^2.2.1",
     "lodash": "^4.17.20",
-    "moment": "^2.28.0",
-    "pako": "^1.0.11",
+    "moment": "^2.29.1",
+    "pako": "^2.0.2",
     "pluralize": "^8.0.0",
     "prop-types": "^15.7.2",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "reactstrap": "^8.6.0",
-    "sass": "^1.3.0",
-    "sass-loader": "^8.0.2",
-    "ts-loader": "^7.0.5",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "reactstrap": "^8.7.1",
+    "sass": "^1.30.0",
+    "sass-loader": "^10.1.0",
+    "ts-loader": "^8.0.12",
     "typescript": "^3.9.7",
-    "webpack": "^4.44.1",
-    "webpack-cli": "^3.3.12",
-    "webpack-fix-style-only-entries": "^0.5.1",
-    "webpack-shell-plugin": "^0.5.0"
+    "webpack": "^5.10.3",
+    "webpack-cli": "^4.2.0",
+    "webpack-remove-empty-scripts": "^0.6.3",
```

Comment:
webpack-fix-style-only-retries hasn't been updated since October. No webpack 5 version available. Switched to webpack-remove-empty-scripts as per recommendation